tutorial/react/getting-started/nesting-data-types/initial/App.tsx

```diff
@@ -11,7 +11,10 @@ export default function App() {
   const publicApiKey = "{% LIVEBLOCKS_PUBLIC_KEY %}";
 
   return (
-    <LiveblocksProvider publicApiKey={publicApiKey}>
+    <LiveblocksProvider
+      publicApiKey={publicApiKey}
+      key={Math.random() /* Fixes a tutorial bug, don't do this */}
```

Comment:
Is there no other way to fix this? Could we trigger a Sandpack refresh/reload when you click on the solution button instead?

Reply:
I'm actually doing that too, just forgot to commit it on the other PR. Doesn't work without the `key` still. I'll mess around a bit and see if I can find a better way.

Reply:
No, I don't see a way around this. This is the only way that isn't buggy, and I prefer this solution to people writing in every few works saying it doesn't work